packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts

```diff
@@ -1,19 +1,62 @@
-import { fromAssetId } from '@shapeshiftoss/caip'
-import { Asset } from '@shapeshiftoss/types'
+import { AssetId, fromAssetId } from '@shapeshiftoss/caip'
+import { Asset, KnownChainIds } from '@shapeshiftoss/types'
 import { AxiosResponse } from 'axios'
 
 import { SwapError, SwapErrorTypes } from '../../../../api'
 import { bn, bnOrZero } from '../../../utils/bignumber'
 import { ZrxPriceResponse } from '../../types'
-import { zrxService } from '../zrxService'
+import { zrxServiceFactory } from '../zrxService'
+
+export const baseUrlFromChainId = (chainId: string): string | undefined => {
+  switch (chainId) {
+    case KnownChainIds.EthereumMainnet: {
+      return 'https://api.0x.org/'
+    }
+    case KnownChainIds.AvalancheMainnet:
+      return 'https://avalanche.api.0x.org/'
+    default:
+      return undefined
+  }
+}
+
+export const usdcContractFromChainId = (chainId: string): string | undefined => {
+  switch (chainId) {
+    case KnownChainIds.EthereumMainnet: {
+      return '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48'
+    }
+    case KnownChainIds.AvalancheMainnet:
+      return '0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e'
+    default:
+      return undefined
+  }
+}
+
+export const isNativeEvmAsset = (assetId: AssetId): boolean => {
+  const { chainId, assetNamespace, assetReference } = fromAssetId(assetId)
+  switch (chainId) {
+    case KnownChainIds.EthereumMainnet: {
+      return assetNamespace === 'slip44' && assetReference === '60'
```

Comment:
could also just compare `assetId === ethAssetId` to use constants over inline string literals

Reply:
I like it. Actioned in https://github.com/shapeshift/lib/pull/850/commits/2efa3a723489ce9ad11cfba9389859ac0e61da58.